src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/StaffReferenceProfileAdvanceIntegrationSearchTest.java

```diff
@@ -117,6 +109,7 @@ void should_return_staff_user_with_status_code_200_when_flag_enabled_default_pag
                 .userType("1")
                 .jobTitle("2")
                 .role("task supervisor,case allocators,staff administrator")
+                .skill("1")
```

Comment:
search string does not have the skill param, include it

Reply:
done